docs/web-servers/lamp/how-to-install-a-lamp-stack-on-debian-10/index.md

```diff
@@ -0,0 +1,308 @@
+        sudo a2ensite example.org
+
+    {{< note >}}
+If you need to disable a site, you can use issue the following command:
+
+    sudo a2dissite example.com
+    {{< /note >}}
+
+1.  Disable the default virtual host to minimize security risks:
+
+        sudo a2dissite 000-default.conf
+
+1.  Restart Apache:
+
+        sudo systemctl restart apache2
+
+
+## MariaDB
+
+MariaDB is a *relational database management system* (RDBMS) and is a popular component of many applications.
+
+### Install MariaDB
+
+1.  Install MariaDB:
+
+        sudo apt-get install mariadb-server
+
+    Input a secure password when prompted by the installation.
+
+1.  Run `mysql_secure_installation` to remove the test database and any extraneous user permissions added during the initial installation process:
+
+        sudo mysql_secure_installation
+
+    It is recommended that you select yes (`y`) for all questions. If you already have a secure root password, you do not need to change it.
+
+### Set Up a MariaDB Database
+
+Next, you can create a database and grant your users permissions to use databases.
+
+1.  Log in to MariaDB:
+
+        sudo mysql -u root -p
+
+    Enter MariaDB's root password when prompted.
+
+1.  Create a database and grant your users permissions on it. Change the database name (`webdata`) and username (`webuser`). Change the password (`password`):
+
+        create database webdata;
+        grant all on webdata.* to 'webuser' identified by 'password';
+
+1.  Exit MariaDB:
+
+        quit
+
+## PHP
+
+PHP makes it possible to produce dynamic and interactive pages using your own scripts and popular web development frameworks.
+
+### Install PHP
+
+PHP 7.3 is the [latest version available](http://php.net/supported-versions.php) and has the longest period of support offered as of this guide's publishing:
+
+1.  Install PHP, the PHP Extension and Application Repository, Apache support, and MySQL support:
+
+        sudo apt install php7.3 libapache2-mod-php7.3 php-mysql
+
+    Optionally, install additional cURL, JSON, and CGI support:
+
+        sudo apt install php-curl php-json php-cgi
+
+
+### Configure PHP
+
+1.  Open `/etc/php/7.3/apache2/php.ini` in your text editor and edit the following values. These settings are optimized for the 2GB Linode:
+
+    {{< file "/etc/php/7.3/apache2/php.ini" ini >}}
+error_reporting = E_COMPILE_ERROR|E_RECOVERABLE_ERROR|E_ERROR|E_CORE_ERROR
+error_log = /var/log/php/error.log
+max_input_time = 30
+
+{{< /file >}}
+
+
+    {{< note >}}
+Ensure that all values are uncommented, by making sure they do not start with a semicolon (**;**).
+{{< /note >}}
+
+1.  Create the log directory for PHP and give ownership to the Apache user (`www-data`):
+
+        sudo mkdir /var/log/php
+        sudo chown www-data /var/log/php
+
+    {{< note >}}
+If you plan on using your LAMP stack to host a WordPress server, install additional PHP modules: `sudo apt install php-curl php-gd php-mbstring php-xml php-xmlrpc`
```

Comment:
I think php-curl is installed on line 237

Reply:
fixed